mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,185 @@
+                              color: Colors.white),
+                          onPressed: () {
+                            _cropController.rotateLeft();
+                          },
+                        ),
+                        IconButton(
+                          icon: const Icon(Icons.rotate_right,
+                              color: Colors.white),
+                          onPressed: () {
+                            _cropController.rotateRight();
+                          },
+                        ),
+                      ],
+                    ),
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: <Widget>[
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_free_rounded,
+                                  color: _selectedAspectRatio == null
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = null;
+                                    _selectedAspectRatio = null;
+                                  })),
+                          const Text('Free',
+                              style: TextStyle(color: Colors.white)),
+                        ],
+                      ),
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_square_rounded,
+                                  color: _selectedAspectRatio == 1
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = 1;
+                                    _selectedAspectRatio = 1;
+                                  })),
+                          const Text('1:1',
+                              style: TextStyle(color: Colors.white)),
+                        ],
+                      ),
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_16_9_rounded,
+                                  color: _selectedAspectRatio == 16 / 9
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = 16 / 9;
+                                    _selectedAspectRatio = 16 / 9;
```

Comment:
I think dart uses doubles for doing this type of division behind the scenes, but  it may be better to use `16.0 / 9.0` just to be explicit. I originally assumed this was integer division and would truncate...

Reply:
Changed those to doubles as well